src/types/model.ts

```diff
@@ -525,14 +608,14 @@ export interface Collaborator {
 export interface Election {
   electionId: number
   electionType: string
-  finalVote: boolean
+  finalVote?: boolean
   status: string
-  createDate: string
-  lastUpdate: string
-  finalVoteDate: string
+  createDate: string | number
```

Comment:
Seems odd it can sometimes be a string or a number.  The response from the backend looks like it's always a number.  Do we create these in other ways?